yucca/training/configuration/configure_paths_and_plans.py

```diff
@@ -0,0 +1,159 @@
+import yucca
+import torch
+from batchgenerators.utilities.file_and_folder_operations import join, isdir, subdirs, maybe_mkdir_p, isfile, load_json
+from dataclasses import dataclass
+from typing import Union
+from yucca.paths import yucca_models, yucca_preprocessed_data
+from yucca.preprocessing.UnsupervisedPreprocessor import UnsupervisedPreprocessor
+from yucca.preprocessing.ClassificationPreprocessor import ClassificationPreprocessor
+from yucca.utils.files_and_folders import recursive_find_python_class
+
+
+@dataclass
+class PathAndPlanConfig:
```

Comment:
This dataclass has become very big. Wouldn't it maybe make sense to split into PathConfig and PlanConfig?

Reply:
Yea I was also thinking about this when I was finishing it up. I'll do that!